backend/src/seeders/seed.data.ts

```diff
@@ -26,7 +26,7 @@ class seedGame {
 
 class seedMessage {
 	sender_id: number;
-	chat_id: number;
+	chat: number;
```

Comment:
ah cool so this is the chat id? same qyestion as earlier.. must say that I liked the id within the chat_id but maybe its meaning changed?

Reply:
the name of the relationship changed so it also changed here